rep/processors/CtGovPdfProcessor.py

```diff
@@ -33,26 +32,26 @@ def _process_blob_wrapper(self, blob, source_url):
         num_list = re.findall(self.VOTE_FOR_PATTERN, page_content)
         year = source_url.split('/')[3]
 
+        # Length of the vote list and/or num_list will be 0 (or 1) if the PDF isn't a vote file that we know
+        # how to read
+        if len(votes) <= 1 or len(num_list) <= 0:
```

Comment:
I think `num_list` isn't the right name for this. It's more like `subject_of_the_vote`, and from what I can see, it should always equal 1. We can also use this instead of `foo` further down